source/adios2/core/VariableBase.cpp

```diff
@@ -265,6 +257,12 @@ void VariableBase::InitShapeType()
     }
 
     /* Extra checks for invalid settings */
+    if (m_DebugMode)
+        CheckDimsCommon("DefineVariable(" + m_Name + ")");
```

Comment:
Like in here, DefineVariable("myvar") is what the user needs to look for. This is a good debugging message as it's non-ambiguous.